src/interpreter/flow.rs

```diff
@@ -104,9 +104,7 @@ where
 
         self.memory[self.registers[REG_FP] as usize..self.registers[REG_SP] as usize].copy_from_slice(stack.as_slice());
 
-        // TODO set balance for forward coins to $bal
-        // TODO set forward gas to $cgas
-
+        self.registers[REG_BAL] = b;
```

Comment:
According to the specs, it should be `d`. Maybe typo there? https://github.com/FuelLabs/fuel-specs/blob/a65e14dfe5c0049ebf4f8e0b33b061c60927d7d9/specs/vm/opcodes.md?plain=1#L1098

Reply:
cc @adlerjohn

Reply:
Good point, looks like a typo in the specs!

Reply:
https://github.com/FuelLabs/fuel-specs/pull/292